tests/sqlparser_sqlite.rs

```diff
@@ -24,6 +24,51 @@ use sqlparser::ast::*;
 use sqlparser::dialect::{GenericDialect, SQLiteDialect};
 use sqlparser::tokenizer::Token;
 
+#[test]
+fn pragma_no_value() {
```

Comment:
Can you please also add a test for the `[schema-name '.'] pragma-name` variant as well? 

like `PRAGMA foo.cache_size = 5`?